Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.8
 
 WORKDIR /opt/project
-
+MOVE . .
```

Comment:
@pranshu667 ???

Reply:
I thought there is a new version of Dockerfile syntax ...
https://docs.docker.com/engine/reference/builder/
nope, it's not
what's the point of this?

Reply:
Oh
That's a spam

https://joel.net/how-one-guy-ruined-hacktoberfest2020-drama